packer/build_image.sh

```diff
@@ -190,31 +192,21 @@ check_deb_exists () {
     done
 }
 
-import_gpg_key () {
```

Comment:
If we are removing this part at all
I think it is not required any more to keep and update the key in this file
https://github.com/scylladb/scylla-machine-image/blob/10adaa1ad2d6796c82927890a81d5ffde2957efd/packer/build_image.sh#L17-L18

Reply:
You are right, removed